pywbem/_valuemapping.py

```diff
@@ -72,36 +73,74 @@ class CIM_Foo {
           };
 
       Assuming this class exists in a WBEM server, the following code will
-      create a value mapping for this property and will print a few property
-      values and their corresponding `Values` strings::
+      create a value mapping for this property and will look up the `Values`
```

Comment:
change will create  to will get the class from the server, create a value mapping.

Reply:
DONE